scripts/userstyles.yml

```
@@ -821,6 +821,14 @@ userstyles:
     categories: [discussion_forum, social_networking]
     color: text
     current-maintainers: [*Guaxinim5573]
+  tidal-web:
+    name: Tidal Web
+    categories: [music]
+    icon: tidal
+    color: sapphire
+    readme:
+      app-link: https://listen.tidal.com
```

```suggestion
    name: Tidal Web
    link: https://listen.tidal.com
    categories: [music]
    icon: tidal
    color: sapphire
```
We recently changed this, sorry.